README.md

```diff
@@ -12,20 +12,23 @@ parts of the [Whitehall](https://github.com/alphagov/whitehall) application.
 
 Not all formats that this app can handle are rendered by it in production.
 
-| Format | Example | Production status |
+| Format | Live example | Production status |
 |---|---|---|
-| Case study | [View example](https://www.gov.uk/government/case-studies/2013-elections-in-swaziland) | Migrated |
+| Case study | [View on GOV.UK](https://www.gov.uk/government/case-studies/2013-elections-in-swaziland) | Migrated |
 | Coming soon | | Rendered by Whitehall |
-| Detailed guide | [View example](https://www.gov.uk/guidance/waste-exemption-nwfd-2-temporary-storage-at-the-place-of-production--2) | Migrated |
-| Document collection | | Rendered by Whitehall |
-| Fatality notice | | Rendered by Whitehall |
+| Consultation | | Rendered by Whitehall |
+| Detailed guide | [View on GOV.UK](https://www.gov.uk/guidance/waste-exemption-nwfd-2-temporary-storage-at-the-place-of-production--2) | Migrated |
+| Document collection | [View on GOV.UK](https://www.gov.uk/government/collections/statutory-guidance-schools) | Migrated |
```

Comment:
Document collection is still rendered by whitehall for draft content (https://github.com/alphagov/whitehall/blob/master/app/models/document_collection.rb#L67) - do we want to note that here? Or are we only concerned about the published editions being rendered by government-frontend?